### PR TITLE
Add docker support

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
@akatsoulas @comzeradd r?

File: bin/run-reclama.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+find ./static -mindepth 1 -not -name '.gitkeep'| xargs rm -rf
+./manage.py collectstatic --noinput
+./manage.py syncdb --noinput
+./manage.py migrate --noinput
```
Comment (comzeradd) [inline bin/run-reclama.sh:5]: We don't need syncdb. migrate is enough since Django 1.7.

**Comment by johngian** [comment]
@comzeradd PR updated

**Comment by comzeradd** [comment]
Looks good. r+ :ice_cream:

**Pull request merged**